catalog/core/catalog-core-directorymonitor/src/main/java/org/codice/ddf/catalog/content/monitor/DurableFileComponent.java

```diff
@@ -32,18 +36,28 @@
                     + " Use the fileName option to set the dynamic expression.");
         }
 
-        File file = new File(remaining);
-
-        DurableFileEndpoint result = new DurableFileEndpoint(uri, this);
-        result.setFile(file);
+        this.remaining = remaining;
+        String isDav = String.valueOf(parameters.get("isDav"));
```

Comment:
❗️ this local variable is hiding the global `isDav` Boolean. Recommend renaming.